packages/react-native-worklets/src/runLoop/uiRuntime/setTimeoutPolyfill.ts

```diff
@@ -7,7 +7,7 @@ export function setupSetTimeout() {
 
   const setTimeoutPolyfill = (
     callback: (...args: unknown[]) => void,
-    delay: number = 1,
+    delay: number = 0,
```

Comment:
The default value of delay is 0 - https://developer.mozilla.org/en-US/docs/Web/API/Window/setTimeout#delay